packages/desktop/src/providers/manager.rs

```diff
@@ -0,0 +1,305 @@
+use std::{
```

Comment:
file should be renamed to `provider_manager.rs` - might be the source of the compile error